index.html

```diff
@@ -1,37 +1,192 @@
+        <h1 class="name">I'm Nabil Glad to see you!</h1>
+        <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project you need coded, don't hestiate to
+            contact me.</p>
+        <p class="touch">LET'S CONNECT</p>
     </section>
-    </nav>
-</header>
-<section class="maintext">
-<h1 class="name">I'm Nabil Glad to see you!</h1>
-<p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-<p class="touch">LET'S CONNECT</p>
-</section>
-<section class="icon">
-    <ul class="list">
-    <li><a href ="https://twitter.com/NeoNabil2"><img class="icons" src="twitter.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://www.linkedin.com/"><img class="icons" src="Linkedinicon.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://medium.com/"><img class="icons" src="medium.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
-    </ul>
-</section>
+    <section class="icon">
+        <ul class="list">
+            <li><a href="https://twitter.com/NeoNabil2"><img class="icons" src="twitter.png" alt="socialmediaicon"></a>
+            </li>
+            <li><a href="https://www.linkedin.com/"><img class="icons" src="Linkedinicon.png" alt="socialmediaicon"></a>
+            </li>
+            <li><a href="https://medium.com/"><img class="icons" src="medium.png" alt="socialmediaicon"></a></li>
+            <li><a href="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a></li>
+            <li><a href="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
+        </ul>
+    </section>
+
+
+    <section class="works">
+        <section class="content">
+            <img class="snapshot" src="SnapshootPortfolio(1).png" alt="picture">
+            <p class="text">Tonic</p>
+            <section class="block">
+                <p class="text1">CANOPY</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
```

Comment:
- [ ] Please group these technologies in a `ul` tag with `li` children. this makes it more semantic since these are actually a list of items.
- [ ] make sure that each is inside an `a` tag as well.
```suggestion
            <ul class="cards">
                <li class="firstcard"><a href="">HTML</a></li>
                <li class="firstcard"><a href="">CSS</a></li>
                <li class="firstcard"><a href="">javascript</a></li>
            </ul>
```